app/views/Settings/ENDebugMenu.js

```diff
@@ -0,0 +1,178 @@
+import React, { useEffect } from 'react';
+import { Alert, BackHandler, ScrollView } from 'react-native';
+
+import { NavigationBarWrapper } from '../../components';
+import { Item } from './Item';
+import { Section } from './Section';
+import {
+  detectExposuresNow,
+  simulateExposure,
+  simulatePositiveDiagnosis,
+  disableExposureNotifications,
+  resetExposureDetectionError,
+  resetLocalExposures,
+  getAndPostDiagnosisKeys,
+  simulateExposureDetectionError,
+} from '../../exposureNotificationsNativeModule';
+
+export const EN_DEBUG_MENU_SCREEN_NAME = 'ENDebugMenu';
+export const EN_LOCAL_DIAGNOSIS_KEYS_SCREEN_NAME = 'ENLocalDiagnosisKeyScreen';
+
+export const ENDebugMenu = ({ navigation }) => {
+  useEffect(() => {
+    const handleBackPress = () => {
+      navigation.goBack();
+      return true;
+    };
+
+    BackHandler.addEventListener('hardwareBackPress', handleBackPress);
+
+    return () => {
+      BackHandler.removeEventListener('hardwareBackPress', handleBackPress);
+    };
+  }, [navigation]);
+
+  const backToSettings = () => {
+    navigation.goBack();
+  };
+
+  const showErrorAlert = (errorString) => {
+    Alert.alert('Error', errorString, [{ text: 'OK' }], { cancelable: false });
+  };
+
+  const showSuccessAlert = (messageString) => {
+    Alert.alert(
+      'Success',
+      messageString,
+      [
+        {
+          text: 'OK',
+        },
+      ],
+      { cancelable: false },
+    );
+  };
+  return (
+    <NavigationBarWrapper title={'EN Debug Menu'} onBackPress={backToSettings}>
+      <ScrollView>
+        <Section>
+          <Item
+            label='Detect Exposures Now'
+            onPress={() => {
+              const cb = (errorString) => {
```

Comment:
theres a lot going on here, I might consider extracting a function something like: `handleOnPressDetectExposures`

you could even be fancy and do something like:
```typescript 

const handleOnPressSimulationButton(callSimulatedEvent, successMessage) => {
   return () => {
       const cb = (errorString) => {
                if (errorString) {
                  showErrorAlert(errorString);
                } else {
                  showSuccessAlert(successMessage)
                }
              };
              callSimulatedEvent(cb);
   }
}


... onPress={handleOnPressSimulationButton(detectExposures, "Exposure detection successful")}
```